cluster/examples/kubernetes/ceph/common.yaml

```
@@ -41,6 +41,10 @@ metadata:
   labels:
     operator: rook
     storage-backend: ceph
+    app.kubernetes.io/name: rook-ceph
+    app.kubernetes.io/instance: rook-ceph
+    app.kubernetes.io/component: csi
```

Instead of `csi`, how about more specific with `ceph-csi` where it's for the csi driver?
```suggestion
    app.kubernetes.io/component: ceph-csi
```